src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -84,6 +84,8 @@
      */
     public static final URLTemplate UPLOAD_SESSION_URL_TEMPLATE = new URLTemplate("files/upload_sessions");
 
+    private static final String CLASSIFICATION_TEMPLATE_KEY = "securityClassification-6VMVochwUWo";
```

Comment:
See note in `BoxFile.java` — we should probably have this constant live in one place